crates/cheatnet/tests/cheatcodes/prank.rs

```diff
@@ -806,25 +516,24 @@ fn prank_all_span() {
 
     let contract_address_1 = test_env.deploy("PrankChecker", &[]);
     let contract_address_2 = test_env.deploy("PrankCheckerLibCall", &[]);
-    let selector = "get_caller_address";
 
     test_env.prank(CheatTarget::All, 123, CheatSpan::Number(1));
 
     assert_success(
-        test_env.call_contract(&contract_address_1, selector, &[]),
+        test_env.call_contract(&contract_address_1, "get_caller_address", &[]),
         &[Felt252::from(123)],
     );
     assert_success(
-        test_env.call_contract(&contract_address_1, selector, &[]),
-        &[ContractAddress(patricia_key!(TEST_ADDRESS)).into_()],
+        test_env.call_contract(&contract_address_1, "get_caller_address", &[]),
+        &[contract_address!(TEST_ADDRESS).into_()],
```

Comment:
[nitpick] Isn't this conversion doable with our `conversions`? Relying on the external ones can be frustrating to update

Reply:
It's not possible atm, at least not in any nicer way.
Best we could do is
```rust
Felt252::from_str_radix(TEST_ADDRESS, 16).unwrap()
```

Other than that, we use `contract_address!` macro in few places already and it's just
```rust
#[macro_export]
macro_rules! contract_address {
    ($s:expr) => {
        ContractAddress(patricia_key!($s))
    };
}
```

Reply:
reasonable